sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -387,7 +387,7 @@
       "get": {
         "summary": "Retrieve a device from the identity registry of an IoT hub.",
         "description": "Get a device from the identity registry of an IoT hub.",
-        "operationId": "RegistryManager_GetDevice",
+        "operationId": "Devices_Get",
```

Comment:
"Devices_GetIdentity" since there is already a "Devices_Get" above

Reply:
In previous revision I thought you agreed to Abhipsa's comment for Devices_Get. But it's fine, I can change this.